statistics/feedback.go

```diff
@@ -687,8 +686,11 @@ func buildNewHistogram(h *Histogram, buckets []bucket) *Histogram {
 type queryFeedback struct {
 	IntRanges []int64
 	// HashValues is the murmur hash values for each index point.
+	// Note that index points will be stored in `IndexPoints`, we keep it here only for compatibility.
 	HashValues  []uint64
```

Comment:
Change `HashValues` to `IndexPoints` won't change the encoded things stored in tikv, and this struct is one thing only in memory. So no compatibility problem if we remove it?

Reply:
If there are old feedbacks that use `HashValues`, the new tidb cannot recognize it.

Reply:
It's a memory structure. After upgrading the TiDB, no old ones exist?

Reply:
It is not only a memory structure, but it is also used to encode and decode feedbacks that stored in tikv.